### DISCUSSION TITLE
How to create missing folders when using :move?

### DISCUSSION THREAD

**Discussion by 4ndv** [helix-editor/helix#12366] [Q&A] [answered]
I often have a need to move some files to folders that are not yet exist. When creating a new file, I can use `:w!` to create all the missing folders, but is there some solution for `:move`?

Right now what I do is run `:sh mkdir -p folders/i/need/to/be/there` before `:move`ing, but that is a bit cumbersome.

Is there any way to do it using single command?

**Comment by kirawi** [comment, marked as answer]
https://github.com/helix-editor/helix/pull/11593

**Reply by 4ndv** [reply]
Hm, looks like this and #11093 is actually the ones I need, subscribed to that, thanks!
